### ISSUE TITLE
Artifact can't be generated

### ISSUE THREAD

**Issue by enriquezrene** [okta/oktasdk-java#11]
Building with maven throw an error, this dependency can't be resolved:

```xml
        <dependency>
            <groupId>com.okta.libraries.framework</groupId>
            <artifactId>aperture-reporter</artifactId>
            <version>${aperture-reporter.version}</version>
        </dependency>
```

**Comment by senthilkumaran-okta** [comment]
@enriquezrene, this is fixed in recent commit c8acbedafeb4ae3187b0c80b668bf816b76f00b3 where we removed an internal dependency you could pull the update and build it again.

**Comment by enriquezrene** [comment]
Yes, I have removed also and now my app is running
